pkgs/by-name/ga/gamma-launcher/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "gamma-launcher";
+  version = "2.5";
+  pyproject = true;
+
+  disabled = python3Packages.pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "Mord3rca";
+    repo = "gamma-launcher";
+    tag = "v${version}";
+    hash = "sha256-qzjfgDFimEL6vtsJBubY6fHsokilDB248WwHJt3F7fI=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
+
+  dependencies = with python3Packages; [
+    beautifulsoup4
+    cloudscraper
+    gitpython
+    platformdirs
+    py7zr
+    python-unrar
+    requests
+    tenacity
+    tqdm
+  ];
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
```

```suggestion
  nativeCheckInputs = [ versionCheckHook ];
```

This is only for `buildPythonPackage`